cmd/aws-janitor-boskos/main.go

```diff
@@ -132,8 +164,12 @@ func cleanResource(res *common.Resource) error {
 		}
 	}
 
-	duration := time.Since(start)
-
-	logrus.WithFields(logrus.Fields{"name": res.Name, "duration": duration.Seconds()}).Info("Finished cleaning")
+	collectMetric(start, res.Name, "clean")
+	logrus.WithFields(logrus.Fields{"name": res.Name, "duration": time.Since(start).Seconds()}).Info("Finished cleaning")
```

Comment:
do you think we should include sweepCount somehow as well? might be interesting to understand if a longer duration is due to multiple sweeps.

Reply:
That sounds a great idea, I will update this PR over the weekend
thanks for your review and feedback

Reply:
added, but not 100% if it is correct, when you have time can you please take a look and let me know? thanks!